state/export_test.go

```diff
@@ -0,0 +1,52 @@
+package state
+
+import (
+	abci "github.com/tendermint/tendermint/abci/types"
+	dbm "github.com/tendermint/tendermint/libs/db"
+	"github.com/tendermint/tendermint/types"
+)
+
+const ValSetCheckpointInterval = valSetCheckpointInterval
+
+// UpdateState is an alias for updateState exported from execution.go,
+// exclusively and explicitly for testing. TODO: Remove.
```

Comment:
Is the TODO only related to this function? or all functions in this package?

Reply:
All functions in this file need to eventually be refactored away. In our tests, we shouldn't be depending on private methods inside packages.

We could have one big TODO for the whole file instead, but when using certain IDEs the IDE pops up with the description of the method. That'll hopefully discourage people from using them.

Reply:
Should we add TODO to every function then?

Reply:
There's already a TODO for each function. Are you asking if we should remove the TODOs and rather just have one in the file somewhere? We can certainly do that, and include a note about why dependence on the functions/constants needs to be removed.